### DISCUSSION TITLE
Welcome to Couenne Discussions!

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/Couenne#49] [General]
## 👋 Welcome!
  As you may have heard, Github has been developing a new feature called Discussions, which is now being rolled 
  out in beta to all projects. We envision Discussions as a way of moving the kinds of conversations that currently 
  take place on the old Mailman list serve directly to where development is taking place! To take this thing for a test 
  drive, please post here when you would otherwise post to the list serve. This is a place to connect with other members 
  of our community on topics not suited for opening an Issue. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by merraksh** [comment]
As one of the original Couenne devs, I'll start. I'm Pietro Belotti and I think Discussions are a very nice addition, especially given that Markdown is supported :-)

Looking forward to lots of good ideas here!

**Comment by ggpolak** [comment]
First, thanks to those who maintain COUENNE. It has been an invaluable resources for solving several MINLP problems that come up in my research. I have used it effectively on rather large problem instances as called from GAMS. 

Now, I have started to call COUENNE from pyomo via spyder and anaconda on my laptop. I have been able to solve some small instances of the MINLP of interest, but the software stalls on slightly larger instances, not even approaching the size that I am able to solve calling COUENNE from GAMS on the same laptop. No messages, feedback, or output of any kind are provided.  Does anyone have any suggestions that would allow me to solve larger instances? Are there options in COUENNE or pyomo commands that could help break through?
